just_audio/android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -134,7 +134,6 @@ private void startWatchingBuffer() {
         handler.post(bufferWatcher);
     }
 
-    @Override
     public void onAudioSessionId(int audioSessionId) {
```

Comment:
The removal of `@Override` looked suspicious as that is there to alert you if the method you're overriding is not in the superclass/interface (in this case `AudioListener`. It looks like it was renamed - can you rename it to `onAudioSessionIdChanged` and add back `@Override`?